classes/controller/AdminController.php

```diff
@@ -3847,7 +3847,7 @@ protected function copyFromPost(&$object, $table)
     {
         /* Classical fields */
         foreach ($_POST as $key => $value) {
-            if (array_key_exists($key, $object) && $key != 'id_' . $table) {
+            if (property_exists($object, $key) && $key != 'id_' . $table) {
```

Comment:
```suggestion
            if (array_key_exists($key, get_object_vars($object)) && $key != 'id_' . $table) {
```
Otherwise it will return true for private and protected